### PR TITLE
Add contributor-milestone-persona.md and Update contributor-domain-persona.md

### PR DESCRIPTION
**Pull request by lishengbao**

### CODE REVIEW

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 1]
```diff
@@ -22,37 +22,100 @@
...
 ## 代码贡献者数量
 
-- 定义：
-- 权重：
-- 阈值：
+- 定义：在过去90天内有多少活跃的代码类贡献者。
+- 权重：16%
+- 阈值：500
 
 ## 代码人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: 代码贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 2]
```diff
@@ -22,37 +22,100 @@
...
 ## Issue贡献者数量
 
-- 定义：
-- 权重：
-- 阈值：
+- 定义：在过去90天内有多少活跃的Issue类贡献者。
+- 权重：16%
+- 阈值：1000
 
 ## Issue人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: Issue贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 3]
```diff
@@ -22,37 +22,100 @@
...
 ## 观察者数量
 
-- 定义：
-- 权重：
-- 阈值：
+- 定义：在过去90天内有多少活跃的star，fork观察者。
+- 权重：8%
+- 阈值：2000
 
 ## 观察者人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: 观察者贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+# 评估模型中的指标
+
+## 核心贡献者数量
+
+- 定义：在过去90天内有多少活跃的核心贡献者数量。
+- 权重：20%
+- 阈值：20
+
+## 核心人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: 核心贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- 权重：30%
+- 阈值：200
+
+## 常客贡献者数量
+
+- 定义：在过去90天内有多少活跃的常客贡献者数量。
+- 权重：12%
+- 阈值：100
+
+## 常客人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: 常客贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+- 阈值：25
+
+  
+## 访客贡献者数量
+
+- 定义：在过去90天内有多少活跃的访客贡献者数量。
+- 权重：8%
+- 阈值：2000
+
+## 访客人均贡献次数
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: 访客贡献次数

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+## 后端
+
+* Yehui Wang
+* Shengbao Li
+* Huatian Qin
+
+## 评估模型
+
+* Yehui Wang
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: Liang Wang

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 8]
```diff
@@ -22,37 +22,100 @@
...
+
+## 后端
+
+* Yehui Wang
+* Shengbao Li
+* Huatian Qin
+
+## 评估模型
+
+* Yehui Wang
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: Liang Wang

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 9]
```diff
@@ -22,37 +22,100 @@
...
+* [开源生态评估与度量的思考（二）——评估体系的多维空间](https://mp.weixin.qq.com/s/ZEcphoxPVTux5_0J7JMxqA) 
+* [开源生态评估与度量的思考（三）——贡献者的动与静](https://mp.weixin.qq.com/s/MDzUmOPgzBRaPMDnKlhaRw) 
+
+# 贡献者
+
+## 前端
+
+* Feng Zhong
+* Xingyou Lai
+
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: Shengxiang Zhang

File: metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md [hunk 10]
```diff
@@ -22,37 +22,100 @@
...
+## 后端
+
+* Yehui Wang
+* Shengbao Li
+* Huatian Qin
+
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-domain-persona.md]: Shengxiang Zhang

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+* [开源生态评估与度量的思考（二）——评估体系的多维空间](https://mp.weixin.qq.com/s/ZEcphoxPVTux5_0J7JMxqA) 
+* [开源生态评估与度量的思考（三）——贡献者的动与静](https://mp.weixin.qq.com/s/MDzUmOPgzBRaPMDnKlhaRw) 
+
+# 贡献者
+
+## 前端
+
+* Feng Zhong
+* Xingyou Lai
+
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: Shengxiang Zhang

File: metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+## 后端
+
+* Yehui Wang
+* Shengbao Li
+* Huatian Qin
+
```
Comment (eyehwan) [inline metrics-models/people/productivity/contributor-persona/contributor-milestone-persona.md]: Shengxiang Zhang

**Review by eyehwan — CHANGES_REQUESTED** [review]
Minor comments

**Comment by lishengbao** [comment]
> Minor comments

done

**Review by eyehwan — APPROVED** [review]

**Pull request merged**